saidump/saidump.cpp

```diff
@@ -10,32 +14,44 @@ extern "C" {
 #include "swss/table.h"
 #include "meta/sai_serialize.h"
 #include "sairediscommon.h"
+#include "swss/json.hpp"
```

Comment:
this is deprecated, use <nlohmann/json.hpp>

Reply:
Fix it.